src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/OpticalDuplicateFinder.java

```diff
@@ -185,6 +188,14 @@ protected final int rapidParseInt(final String input) {
         return val;
     }
 
+    public boolean isOpticalDuplicate(PhysicalLocation a, PhysicalLocation b) {
+      return a.getReadGroup() == b.getReadGroup() &&
+          a.getTile() == b.getTile() &&
+          // There used to not be an abs here, not sure why.
```

Comment:
I would be careful about the ```abs``` since we may not want this method to be "commutative".  Please take a closer look where this is used.